### PR TITLE
Sort revisions based on their generation

### PR DESCRIPTION
**Pull request by evidolob**
This PR adds comparator for knative tree. 
It is sorting revisions nodes based on their `serving.knative.dev/configurationGeneration` label value.
Also highlight revisions which doesn't has traffic. 
And fix tag rendering, add space between labels.
Demo:

Screenshot 2021-03-11 at 09 55 47

Fix: #12

### CODE REVIEW

**Review by lstocchi — APPROVED** [review]
LGTM

File: src/main/java/com/redhat/devtools/intellij/knative/kn/KnConstants.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.intellij.knative.kn;
+
+public interface KnConstants {
```
Comment (jeffmaury) [inline src/main/java/com/redhat/devtools/intellij/knative/kn/KnConstants.java:13]: I think proper and Sonar way is to use final class for that

File: src/main/java/com/redhat/devtools/intellij/knative/tree/KnRevisionDescriptor.java [hunk 2]
```diff
@@ -48,27 +53,35 @@ protected void update(@NotNull PresentationData presentation) {
...
-            presentation.addText(percent + "%", SimpleTextAttributes.GRAYED_ITALIC_ATTRIBUTES);
-            presentation.addText(" " + tag.toString(), SimpleTextAttributes.GRAY_ATTRIBUTES);
 
+        }
 
+        if (Strings.isNullOrEmpty(errorMessage)) {
+            presentation.setTooltip("Revision: " + element.getName());
+            presentation.addText(element.getName(), percent != 0 ? SimpleTextAttributes.REGULAR_ATTRIBUTES : WARNING_ATTRIBUTES);
+        } else {
+            presentation.setTooltip(errorMessage.length() >= 100 ? errorMessage.substring(0, 100) + "..." : errorMessage);
```
Comment (jeffmaury) [inline src/main/java/com/redhat/devtools/intellij/knative/tree/KnRevisionDescriptor.java:76]: I think this kind of logic should be extracted in a separate class utility as it's likely to be used elsewhere

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**